cmd/rad/cmd/resourceprovider.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2023 The Radius Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(resourceProviderCmd)
+	resourceProviderCmd.PersistentFlags().StringP("workspace", "w", "", "The workspace name")
+}
+
+func NewResourceProviderCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "resourceprovider",
+		Short: "Manage resource providers",
```

Comment:
is resource provider a bit too long? could we use a shorter word like how we use env for environment?

Reply:
Clear > Concise. We could also add `rp` as an alias. 

I'm actually going to log a bug for `rad env`. We should define `rad environment` as the command name, and use `rad env` as an alias.

Reply:
https://github.com/radius-project/radius/issues/8043

Reply:
@Reshrahim - I'm adding `rp` as an alias here, and `rt` as an alias for `resource-types`. Speak now if you object 😆